android/app/src/main/java/app/organicmaps/sound/TtsPlayer.java

```diff
@@ -238,6 +238,16 @@ public static void setEnabled(boolean enabled)
     nativeEnableTurnNotifications(enabled);
   }
 
+  public static boolean getAnnounceStreets()
```

Comment:
Are these wrappers really needed? Can Config be called directly?

Reply:
@biodranik just seems like an existing convention; will need to include Config into NavigationService.